### PR TITLE
[ET-1064] Display TC pages with post status

### PR DESCRIPTION
**Pull request by juanfra**
### 🎫 Ticket

[ET-1064] 

### 🗒️ Description

- Adding reference to our TC pages on the page list.

### 🎥 Artifacts 

https://www.loom.com/share/5e22fab6a68f464eb3db253013e0bbe9

### ✔️ Checklist
- [ ] I've included a changelog entry both in readme.txt and changelog.txt files. 
- [ ] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1064]: https://theeventscalendar.atlassian.net/browse/ET-1064

### CODE REVIEW

File: src/Tickets/Commerce/Hooks.php [hunk 1]
```diff
@@ -394,4 +397,26 @@ public function filter_js_include_checkout_url( $urls ) {
...
+	 * @since TBD
+	 *
+	 * @param array   $post_states An array of post display states.
+	 * @param WP_Post $post        The current post object.
+	 */
+	public function add_display_post_states( $post_states, $post ) {
+
+		if ( tribe( Checkout::class )->get_page_id() === $post->ID ) {
+			$post_states['tec_tickets_commerce_page_checkout'] = __( 'Tickets Commerce Checkout Page', 'event-tickets' );
+		}
```
Comment (bordoni) [inline src/Tickets/Commerce/Hooks.php:413]: Move this one into a method in the actual Checkout Class and call it here, so we dont have any logic on on Hooks class.

File: src/Tickets/Commerce/Hooks.php [hunk 2]
```diff
@@ -394,4 +397,26 @@ public function filter_js_include_checkout_url( $urls ) {
...
+
+		if ( tribe( Success::class )->get_page_id() === $post->ID ) {
+			$post_states['tec_tickets_commerce_page_success'] = __( 'Tickets Commerce Success Page', 'event-tickets' );
+		}
```
Comment (bordoni) [inline src/Tickets/Commerce/Hooks.php:417]: Move this one into a method in the actual Success Class and call it here, so we dont have any logic on on Hooks class.

**Review by bordoni — CHANGES_REQUESTED** [review]
Just a minor thing, so that we avoid logic on the Hooks class.

File: src/Tickets/Commerce/Checkout.php [hunk 3]
```diff
@@ -159,4 +159,21 @@ public function get_registration_url() {
...
 	}
+
+	/**
+	 * Maybe add a post display state for special Tickets Commerce Checkout Page in the page list table.
+	 *
+	 * @since TBD
+	 *
+	 * @param array   $post_states An array of post display states.
+	 * @param WP_Post $post        The current post object.
+	 */
```
Comment (paulmskim) [inline src/Tickets/Commerce/Checkout.php:170]: add return to docblock

File: src/Tickets/Commerce/Hooks.php [hunk 4]
```diff
@@ -394,4 +397,21 @@ public function filter_js_include_checkout_url( $urls ) {
...
 	}
+
+	/**
+	 * Add a post display state for special Event Tickets pages in the page list table.
+	 *
+	 * @since TBD
+	 *
+	 * @param array   $post_states An array of post display states.
+	 * @param WP_Post $post        The current post object.
+	 */
```
Comment (paulmskim) [inline src/Tickets/Commerce/Hooks.php:408]: add return to docblock

File: src/Tickets/Commerce/Success.php [hunk 5]
```diff
@@ -122,4 +122,21 @@ public function parse_request() {
...
 	}
+
+	/**
+	 * Maybe add a post display state for special Tickets Commerce Success Page in the page list table.
+	 *
+	 * @since TBD
+	 *
+	 * @param array   $post_states An array of post display states.
+	 * @param WP_Post $post        The current post object.
+	 */
```
Comment (paulmskim) [inline src/Tickets/Commerce/Success.php:133]: add return to docblock

**Review by paulmskim — APPROVED** [review]

**Pull request merged**